### PR TITLE
i18n integration

### PR DESCRIPTION
**Pull request by razbakov**
As a Visitor, I want to switch languages to EN, DE, UK, and RU so that I can understand what the page says in different languages. Currently, only the home page. See https://trello.com/c/qVcr45X5/317-i18n-integration-2

## Acceptance Criteria
- [x] localization is not part of this story, translation will be done later by translators
- [x] all text on the home page is internationalized
- [x] translation files for EN, DE, UK, and RU is present
- [x] when I change value in language selector language of the page changes
- [x] auto-detection of browser language

## Demo
https://deploy-preview-41--wehelpvip.netlify.app/

### CODE REVIEW

**Comment by razbakov** [comment]
@allend13 here is internationalization half way integrated

**Comment by remklov** [comment]
@razbakov I looked into the translation issue. The general implementation seems right, if i use a translation in the template with `{{ $t("exchange") }}` it works. The menu items are not translated though. The reason seems to be, that the `useI18n()` returns an object which is not reactive, so it doesnt get updated when the language changes. 

I got it to work with `{ name: computed(() => t('exchange')), to: '/exchange' }`, but this adds quotation marks in the string. I need to look into it in more detail to find a better solution, but maybe this info helps you to build up on.

**Comment by razbakov** [comment]
@VOlekh See last comment, it should solve the issue. Can you please take it over?

**Comment by razbakov** [comment]
@amanuela97 
- this branch has conflicts that must be resolved
- auto-detection of browser language

**Comment by razbakov** [comment]
@amanuela97 This branch has conflicts that must be resolved

File: components/block/team.vue [hunk 1]
```diff
@@ -37,49 +37,62 @@
...
+import { useI18n } from 'vue-i18n'
+
+export default {
+  setup() {
+    const { t } = useI18n()
+    // eslint-disable-next-line no-unused-vars
+    const people = [
+      {
+        name: 'Gulnara Mustafaeva',
+        role: computed(() => t('home.team.roles.partnerships')),
```
Comment (razbakov) [inline components/block/team.vue:59]: This field will come from API, we can't use computed field here

**Pull request merged**